libpod/logs/log.go

```diff
@@ -101,11 +101,14 @@ func getTailLog(path string, tail int) ([]*LogLine, error) {
 			if err != nil {
 				if errors.Cause(err) == io.EOF {
 					inputs <- []string{leftover}
-					close(inputs)
```

Comment:
Is this close done somewhere else?  Is it not needed?

Reply:
It is closed below